pylossless/pipeline.py

```diff
@@ -796,12 +796,9 @@ def find_outlier_chs(self, inst, picks="eeg"):
         """
         # TODO: Reuse _detect_outliers here.
         logger.info("🔍 Detecting channels to leave out of reference.")
-        if isinstance(inst, mne.Epochs):
-            epochs = inst
-        else:
-            raise TypeError(
+        if epochs is None:
+            epochs = self.get_epochs(rereference=False, picks=picks)
                 "inst must be an instance of mne.Epochs," f" but got {type(inst)}."
```

Comment:
Oups! I think I left a line that was meant to be removed!

```suggestion
                "inst must be an instance of mne.Epochs," f" but got {type(inst)}."
```

Reply:
Already fixed in [2a747c2](https://github.com/lina-usc/pylossless/pull/213/commits/2a747c2320931b32697ffed079e870c9c971f9ee) 🙂